pkg/query-service/tests/docker.go

```diff
@@ -47,9 +47,24 @@ func getCmd(args ...string) *exec.Cmd {
 	return cmd
 }
 
-func startMinio() error {
+func startNetwork(name string) error {
+	cmd := getCmd("docker", "network", "inspect", name)
+	if err := cmd.Run(); err != nil {
+		cmd = getCmd("docker", "network", "create", name)
+		if err := cmd.Run(); err != nil {
+			return fmt.Errorf("failed to create network '%s': %v", name, err)
+		}
+		log.Printf("Network '%s' created", name)
+	} else {
+		log.Printf("Network '%s' already exists", name)
+	}
+	fmt.Println("Network created")
```

Comment:
log.Printf ?

Reply:
yeah that would be better, thank you @nityanandagohain

Reply:
fixed: [ff24346](https://github.com/SigNoz/signoz/pull/6098/commits/ff24346408473b8f6c9d432c119b92ccf0b6051e)